src/input/src/lib.rs

```diff
@@ -79,6 +85,8 @@ pub enum Motion {
     MouseRelative(f64, f64),
     /// x and y in scroll ticks.
     MouseScroll(f64, f64),
+    /// joystick axis position as a normalized float (-1.0 to 1.0)
```

Comment:
Some APIs might use positional sensors through the joystick API. There is no guarantee that the value stays between (-1.0 to 1.0). Could change the comment to "usually in range [-1.0, 1.0]". Use square brackets to tell that the range is including -1.0 and 1.0.